diff --git a/mumbleBot.py b/mumbleBot.py
index d7a434f..1dab61f 100644
--- a/mumbleBot.py
+++ b/mumbleBot.py
@@ -458,6 +458,7 @@ class MumbleBot:
         raw_music = ""
         while not self.exit and self.mumble.is_alive():
 
+            print self.mumble
             while self.is_ffmpeg_running() and self.mumble.sound_output.get_buffer_size() > 0.5 and not self.exit:
                 # If the buffer isn't empty, I cannot send new music part, so I wait
                 self._loop_status = f'Wait for buffer {self.mumble.sound_output.get_buffer_size():.3f}'
